src/ms-store-badge.ts

```diff
@@ -400,57 +402,61 @@
 
   static createSupportedLanguages(): SupportedLanguage[] {
     let languageMap = new Map<string, string>();
-    languageMap.set("Arabic", "ar");
```

Comment:
For backwards compatibility, we still need to support the old language names, right? Suppose there's an existing user who set his <ms-store-badge> language to `be` - would this change break his badge?